include/aws/crt/Types.h

```diff
@@ -45,7 +45,8 @@ namespace Aws
         namespace Io
         {
             using SocketOptions = aws_socket_options;
-        }
+            using IoStream = std::basic_iostream<char, std::char_traits<char>>;
```

Comment:
Any reason to not just use `std::basic_istream`? We're never writing to this thing.

Reply:
Probably not.  We'll eventually need an output stream as well, but this can be changed.